test/props/prop-set.js

```diff
@@ -112,6 +112,9 @@ describe('PropSet', () => {
       assert.strictEqual(set.def.props.g.value, 'green')
       assert.strictEqual(set.def.props.h.value, 'green')
     })
+    it('resolves aliases set inside of alias objects', () => {
+      assert.strictEqual(set.def.props.j.value, 'green')
```

Comment:
that's a great question. it could end up in rawData (actually wondering if it alread shows up in raw data).

Otherwise it could be passed down as key in the prop. I'd suggest the entire object passed down, so outcome would be 

```yaml
props:
  FOO: red
  .alias:
    value: red
    otherStuff: blahblah
    .alias:
      value: red
      moreStuff: howdy
```

This way you could even traverse the alias trail.